### PR TITLE
[doc] explain avg in function_score better

### PR DESCRIPTION
**Pull request by brwe**
relates to #19068

### CODE REVIEW

File: docs/reference/query-dsl/function-score-query.asciidoc [hunk 1]
```diff
@@ -83,9 +83,16 @@ First, each document is scored by the defined functions. The parameter
 `max`::         maximum score is used
 `min`::         minimum score is used
 
-Because scores can be on different scales (for example, between 0 and 1 for decay functions but arbitrary for `field_value_factor`) and also because sometimes a different impact of functions on the score is desirable, the score of each function can be adjusted with a user defined `weight` (). The `weight` can be defined per function in the `functions` array (example above) and is multiplied with the score computed by the respective function.
+Because scores can be on different scales (for example, between 0 and 1 for decay functions but arbitrary for `field_value_factor`) and also
+because sometimes a different impact of functions on the score is desirable, the score of each function can be adjusted with a user defined
+`weight` (). The `weight` can be defined per function in the `functions` array (example above) and is multiplied with the score computed by
```
Comment (clintongormley) [inline docs/reference/query-dsl/function-score-query.asciidoc]: Why the `()`?

File: docs/reference/query-dsl/function-score-query.asciidoc [hunk 2]
```diff
@@ -83,9 +83,16 @@ First, each document is scored by the defined functions. The parameter
...
+the respective function.
 If weight is given without any other function declaration, `weight` acts as a function that simply returns the `weight`.
 
+In case `score_mode` is set to `sum` the individual scores will be combined by a **weighted** average.
```
Comment (clintongormley) [inline docs/reference/query-dsl/function-score-query.asciidoc]: set to `avg`, not `sum`

**Comment by clintongormley** [comment]
Couple of typos otherwise LGTM

**Comment by brwe** [comment]
@clintongormley thanks for the review! pushed a commit.

**Comment by clintongormley** [comment]
LGTM

**Pull request merged**